docs/spark/tutorials/databricks-deployment.md

```diff
@@ -115,7 +115,7 @@ You can use `set JAR` or `spark-submit` to submit your job to Databricks.
 
 3. Set the parameters appropriately.
 
-   ```
+   ```console
```

Comment:
I'm wondering if this should be in a table instead. @mamccrea can you comment?

Reply:
Thanks @mairaw, yes this should be in a table - we're currently reworking these deployment tutorials.